### ISSUE TITLE
Wont start on 3 different 

### ISSUE THREAD

**Issue by SleepyPwasnotavailable** [fesch/Structorizer.Desktop#1070]
Hey there,
already used structorizer like 2 years ago once and everything went flawless,
Today i tryed downloading it again on 3 different machines, all with the newest java updated and non of then can start it.
If i try i get the structorizer pop up with "starting application" and then it closes again and nothing else happens.

**Comment by codemanyak** [comment]
@SleepyPwasnotavailable 
Sorry for your troubles. It would help us a lot if you could give us some more information:
1. What OS are you working on?
2. What kind of Structorizer installation did you perform (Windows installer, Zip package, debian package)?
3. Does "newest Java updated" mean ≥ version 11 (which has been required for almost a year now)? Or just the newest update for Java 8 (which would explain all your troubles)? See [News](https://structorizer.fisch.lu/index.php?include=news) from 2021-09-19 and [Installation Guide](https://help.structorizer.fisch.lu/index.php?menu=45).
4. Could you please download the zipped product for your operating system (Windows/Linux), unpack it and run the appropriate script (Structorizer.bat or structorizer.sh) at the command prompt and send us the error output?
5. Have you still installed an older Structorizer that is running?

**Comment by codemanyak** [comment]
> If I try I get the structorizer pop up with "starting application" and then it closes again and nothing else happens.

Okay, this looks like the Windows installer. I overlooked it. So questions 1 and 2 seem answered...

**Comment by SleepyPwasnotavailable** [comment]
Thanks for responding so quick,

Yea its win 10 and i used the installer,
also i just checked and im running Java 8 Update 341 (build 1.8.0_341-b10) thats probably why i wont work.
I'll upgrade to 11 or higher and then check again.

**Comment by SleepyPwasnotavailable** [comment]
i guess thats why, when i run the .bat, it just closes aswell and dose not give me an error

**Comment by codemanyak** [comment]
> I'll upgrade to 11 or higher and then check again.

Yes, that will do but please read the installation hints in the User Guide if you plan to install an OpneJDK. (I can recommend Adoptium.) With an Oracle Java it should be straightforward if you can live with the new license conditions.

> when i run the .bat, it just closes aswell and dose not give me an error

Well, I asked you to run it from the command prompt, i.e. in the "MSDOS" console (type "cmd" in the start menu). Otherwise (e.g. with double-click from file explorer) you will not of course see the eror message because the Structorizer.bat process dies and does not transfer its standard and error output channels to the Windows desktop.

**Comment by SleepyPwasnotavailable** [comment]
Works with >=11
 Tx guys

**Comment by codemanyak** [comment]
Then go ahaid, have fun. Thank you for the feedback. If you happen to bump into further problems, you know where to get help :-)
